SETUP/tests/phpunit_test_helpers.inc

```diff
@@ -16,8 +16,9 @@ function create_test_user($username)
             SET id = '%1\$s',
                 real_name = '%1\$s',
                 username = '%1\$s',
-                email = '%1\$s@localhost'
-        ", DPDatabase::escape($username));
+                email = '%1\$s@localhost',
+                date_created = %2\$d
+        ", DPDatabase::escape($username), time() - ($age * 86400));
```

Comment:
Just trying to understand the logic here -- for purposes of the default, `date_created` will always be the current timestamp? How is `$age` determined in a way such that it would be greater than zero?

Reply:
Good point. I made this several months ago, my comment above was the wrong way round in that it is to test that an old user who has been on the site more than 21 days cannot access projects reserved for new users. An old user is made by giving the age parameter a value of 22.

Reply:
Will it be used in that way? If so, then is "create" a good description of the function? Also, if so, maybe add comments to clarify?

Reply:
I'll do that.